README.md

```diff
@@ -1,48 +1,54 @@
-# Bio-Formats
+# OME Common C++
 
-[![Build Status](https://travis-ci.org/openmicroscopy/bioformats.png)](http://travis-ci.org/openmicroscopy/bioformats)
```

Comment:
In terms of Travis testing (does not need to happen now), is there anything that can happen at the level of this standalone repository?